frontend/controller/serviceworkers/push.js

```diff
@@ -97,7 +149,18 @@ export default (sbp('sbp/selectors/register', {
           // If the subscription has changed, report it to the server
           pubsub.socket.send(createMessage(
             REQUEST_TYPE.PUSH_ACTION,
-            { action: PUSH_SERVER_ACTION_TYPE.STORE_SUBSCRIPTION, payload: subscriptionInfo }
+            {
+              action: PUSH_SERVER_ACTION_TYPE.STORE_SUBSCRIPTION,
+              payload: {
+                applicationServerKey: applicationServerKey
+                  ? Buffer.from(applicationServerKey).toString('base64')
+                  : null,
+                settings: {
+                  heartbeatInterval: 12 * 60 * 60 * 1000
+                },
```

Comment:
If I'm not mistaken, this requires #2546 to be merged first for it to work, right?

Reply:
This is here to reduce future merge conflicts. It won't do anything in this PR.